src/simulation/ApplyLoad.cpp

```diff
@@ -587,12 +623,37 @@ ApplyLoad::benchmark()
                 mNumAccounts, 0, lm.getLastClosedLedgerNum() + 1, it->first, 1,
                 std::nullopt);
         }
-        else
+        else if (mMode == ApplyLoadMode::SOROBAN)
         {
             tx = mTxGenerator.invokeSorobanLoadTransactionV2(
                 lm.getLastClosedLedgerNum() + 1, it->first, mLoadInstance,
                 mDataEntryCount, mDataEntrySize, 1'000'000);
         }
+        else
+        {
+            if (it->first % 2 == 0)
```

Comment:
Nit: Brief comment about mixing load into thirds would be helpful here, as it's not super clear initially with dereferenced it.